docs/registering-samples.md

```diff
@@ -5,20 +5,22 @@ title: Registering Samples
 
 In the ARGO Data Platform, clinical and molecular data objects are assigned ARGO Identifiers (**ARGO IDs**) used to track the data through the Platform. Each **Donor**, **Specimen**, and **Sample** entity will be assigned an **ARGO ID**, mapping to your program's internal identifiers.
 
-It is important that the relationships between entities are maintained across submissions, as they are fundamental to data tracking and integrity across the ARGO Data Platform.
+It is important that the relationships between entities are maintained across all data submissions and formats, as they are fundamental to data integrity across the ARGO Data Platform.
 
-> Registration is the first step in the data submission life cycle.  You must registers samples before any clinical or molecular data submissions.
+> Registration is the first step in the data submission life cycle.  You **must** registers samples before any clinical or molecular data submissions.
 
 The basic set of data that must be registered for each sample consists of:
-* Submitter Donor ID
-* Donor Gender
-* Submitter Specimen ID
-* Specimen Tissue Source
-* Specimen Tumor or Normal Designation
-* Submitter Sample ID
-* Sample Type
-
-During sample registration, **ARGO IDs** will be assigned to your program data. Any attempts to submit data that does not refer to a previously registered samples will result in an error. You will be prompted to complete sample registration before any clinical or molecular data is submitted.
+* `program_id`
+* `submitter_donor_id`
+* `gender`
+* `submitter_specimen_id`
+* `specimen_tissue_source`
+* `tumour_normal_designation`
+* `specimen_type`
+* `submitter_sample_id`
+* `sample_type`
+
+During sample registration, **ARGO IDs** will be assigned to your program data. Any attempts to submit data that does not refer to a  registered samples will result in an error. You will be prompted to complete sample registration before any clinical or molecular data is submitted.
```

Comment:
extra space between  "a" and "registered". remove "s" from "a registered samples"

"to your program data" could be more specific - "to your donors, samples and specimens"